src/components/molecules/OcTable/GhostElement/GhostElement.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div id="ghost-element" class="ghost-element">
+    <oc-icon :name="firstPreviewIcon" size="large" />
+    <span class="badge">{{ itemCount }}</span>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "GhostElement",
+  props: {
+    previewItems: {
+      type: Array,
+      required: true,
+    },
+  },
+  computed: {
+    firstPreviewIcon() {
+      const icon = this.previewItems[0].icon
+      return icon ? icon : "file"
+    },
+    itemCount() {
+      return this.previewItems.length
+    },
+  },
+}
+</script>
+
+<style lang="scss">
+.ghost-element {
+  background-color: transparent;
+  padding-top: 4px;
+  padding-left: 5px;
+  z-index: 99999;
```

Comment:
There's a variable for this z-index. I don't remember if it came from UiKit or from our design tokens, though. Please check if you can use that variable for this, if you can find it.

Reply:
cc @pascalwengerter

Reply:
Naming isn't good, it's `--oc-z-index-modal` as found in https://github.com/owncloud/owncloud-design-system/blob/master/src/tokens/ods/z-index.yaml @paulcod3 feel free to add a new one for the GhostElement (or a generic one maybe?)

Reply:
Plus we don't seem to use it in both ODS and web 🤔

Reply:
--oc-z-index-modal works fine

Reply:
Yeah it works, but it's semantically incorrect. Imagine someone setting a custom `z-index-modal` through theming (for whatever reason) and suddenly the Ghost element is affected as well ;)

Reply:
FYI I'd be fine with just finding a better name for that variable and rename it.